include/aikido/trajectory/util.hpp

```diff
@@ -45,11 +45,13 @@ UniqueInterpolatedPtr concatenate(
 ///
 /// \param[in] traj Input trajectory
 /// \param[in] referenceState Reference state
+/// \param[out] distance Distance to the closest state
```

Comment:
Feel free to defer to another PR but docstrings should end with a period.

Reply:
Will defer to another PR.